### DISCUSSION TITLE
cant figure out file structure

### DISCUSSION THREAD

**Discussion by slightlysketch** [julianschill/klipper-led_effect#207] [Q&A] [answered]
so i used kiauh to install klipper on 3 machines.
ive installed a ercf/happy hare on one so i understand how to set the install script up but i cant seem to make it work,
![Capture](https://github.com/julianschill/klipper-led_effect/assets/132846113/d158bf71-5e3a-4655-9d9e-652f57e2363d)
i cant seem to figure out the correct install paths for the following script.

./install-led_effect.sh -u [-k <klipper path>] [-s <klipper service name>] [-c <configuration path>]

can anyone help me out on this, im pulling my hair out and already put my fist thru one of my extra monitors.

**Comment by slightlysketch** [comment]
also no matter what i do the -c never takes and always tries to install at "/home/aaron/printer_data/config"

and i get the following message.
"/home/aaron/printer_data/config" does not exist. 
Falling back to /home/aaron/klipper_config as default."

**Comment by slightlysketch** [comment]
./install-led_effect.sh -k/home/aaron/klipper -s/home/aaron/sv06_data -c/home/aaron/sv06_data/config

this is what im tryig to run.

**Comment by slightlysketch** [comment]
any help would be cool
id like to use the led i got but honestly will just run the original non barf one if no one can help.
always hated being poor and only running a single pi for multiple printers.
always made this stuff so much harder.

**Comment by julianschill** [comment, marked as answer]
Try this:
```
./install-led_effect.sh -k /home/aaron/klipper -s klipper-sv06 -c /home/aaron/sv06_data/config
```

Note the spaces before the paths.

**Comment by slightlysketch** [comment]
youre a god and king among men.

thank you,

**Comment by julianschill** [comment]
So it worked? You might have to restart all klipper instances.
